modules/govuk/manifests/apps/ckan.pp

```diff
@@ -0,0 +1,84 @@
+# == Class: govuk::apps::ckan
+##
+# === Parameters
+#
+# [*enabled*]
+#   Should the app exist?
+#
+# [*port*]
+#   What port should the app run on?
+#
+# [*db_password*]
+#   The password for the ckan postgresql role
+#
+class govuk::apps::ckan (
+  $enabled = false,
+  $port    = '3108',
+  $db_hostname = undef,
+  $db_username = 'ckan',
+  $db_password = 'foo',
+  $db_name = 'ckan_production',
+  $redis_host = undef,
+  $redis_port = undef,
+  $secret     = undef,
```

Comment:
Should these be documented in the parameters docs above? It's not obvious to me what `secret` is for.